cli/src/athenapdf.js

```diff
@@ -47,8 +48,16 @@ if (!uriArg) {
     process.exit(1);
 }
 
+//Handle stdin
+if (uriArg === '-')
+{
```

Comment:
Sorry, this is me being pedantic. Since we're already following K&R style braces, can we modify this to follow it? Other than that, it looks good to me.